app/components/rdf-input-fields/mandataris-mandaat-selector.js

```diff
@@ -0,0 +1,69 @@
+import { guidFor } from '@ember/object/internals';
+import { tracked } from '@glimmer/tracking';
+import InputFieldComponent from '@lblod/ember-submission-form-fields/components/rdf-input-fields/input-field';
+import { triplesForPath } from '@lblod/submission-form-helpers';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { NamedNode } from 'rdflib';
+import { replaceSingleFormValue } from 'frontend-lmb/utils/replaceSingleFormValue';
+import { EXT } from 'frontend-lmb/rdf/namespaces';
+
+export default class MandatarisMandaatSelector extends InputFieldComponent {
+  inputId = 'input-' + guidFor(this);
+
+  @service store;
+
+  @tracked mandaat = null;
+  @tracked initialized = false;
+  @tracked bestuursorganen = [];
+
+  constructor() {
+    super(...arguments);
+    this.load();
+  }
+
+  async load() {
+    await this.loadProvidedValue();
+    await this.loadBestuursorgaan();
+    this.initialized = true;
+  }
+
+  async loadBestuursorgaan() {
```

Comment:
This is code is already present in mandataris-fractie-selector, might by worth it to extract this, since we will probably need it again somewhere.

Reply:
or maybe use some inheritance?